### PR TITLE
[hi] Localize pod.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/pod.md` to Hindi `content/hi/docs/reference/glossary/pod.md`

Fixes :- #35600

### CODE REVIEW

**Comment by bishal7679** [comment]
/assign @divya-mohan0209

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/pod.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - core-object
+  - fundamental
+---
+
+सबसे छोटा और सरल कुबेरनेट्स ओब्जेक्ट। पोड आपके क्लस्टर में चल रहे {{< glossary_tooltip text="कंटेनरों" term_id="container" >}} के एक सेट का प्रतिनिधित्व करता है।
+
+<!--more-->
+
+एक पोड आमतौर पर एक प्राथमिक कंटेनर चलाने के लिए स्थापित किया जाता है। यह वैकल्पिक साइडकार कंटेनर भी चला सकता है जो लॉगिंग जैसे पूरक सुविधाओं को जोड़ता है। पॉड्स को आमतौर पर एक {{< glossary_tooltip term_id="deployment" >}} द्वारा प्रबंधित किया जाता है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/pod.md:19]:
```suggestion
एक पॉड आमतौर पर एक प्राथमिक कंटेनर चलाने के लिए स्थापित किया जाता है। यह वैकल्पिक साइडकार कंटेनर भी चला सकता है जो लॉगिंग जैसे पूरक सुविधाओं को जोड़ता है। पॉड्स को आमतौर पर एक {{< glossary_tooltip term_id="deployment" >}} द्वारा प्रबंधित किया जाता है।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from my side, hope you find them useful. Thanks

**Comment by Babapool** [comment]
The changes LGTM (not an official 'hi' localization reviewer/approver). It would be great if you could squash your commits into one for getting the PR ready to merge. Thanks.

**Comment by Babapool** [comment]
Hi @bishal7679 , hope you are doing well. Whenever you are free can you please squash the commits to  make the PR ready to merge. Thanks

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

File: content/hi/docs/reference/glossary/pod.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - core-object
+  - fundamental
+---
+
+सबसे छोटी और सरल कुबेरनेट्स वस्तु। पॉड आपके क्लस्टर में चल रहे {{< glossary_tooltip text="कंटेनरों" term_id="container" >}} के समूह का प्रतिनिधित्व करता है।
+
+<!--more-->
+
+एक पॉड आमतौर पर एक प्राथमिक कंटेनर चलाने के लिए स्थापित किया जाता है। यह वैकल्पिक साइडकार कंटेनर भी चला सकता है जो लॉगिंग जैसे पूरक सुविधाओं को जोड़ता है। पॉड्स को आमतौर पर एक {{< glossary_tooltip term_id="deployment" >}} द्वारा प्रबंधित किया जाता है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/pod.md:19]:
```suggestion
एक पॉड आमतौर पर एक प्राथमिक कंटेनर चलाने के लिए स्थापित किया जाता है। यह वैकल्पिक साइडकार कंटेनर भी चला सकता है जो लॉगिंग जैसे पूरक सुविधाओं को जोड़ता है। पॉड्स को आमतौर पर एक {{< glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment" >}} द्वारा प्रबंधित किया जाता है।
```

**Review by Babapool — COMMENTED** [review]
A small nit, otherwise LGTM:

**Review by Babapool — COMMENTED** [review]
Reupping my LGTM from before. Thanks!
/lgtm

**Pull request merged**